java/ql/lib/semmle/code/java/security/ArithmeticCommon.qll

```diff
@@ -210,7 +178,6 @@ predicate overflowSink(ArithExpr exp, VarAccess use) {
     exp instanceof PostIncExpr or
     exp instanceof MulExpr
   ) and
-  not guardedLessThanSomething(use) and
```

Comment:
Why is this removed (and the line below)?

Reply:
It's part of the barrier, so it's superfluous to filter like this in the sink definition.